lib/karafka/lago_monitor.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Karafka::LagoMonitor < ::Karafka::Instrumentation::Monitor
+  TRACEABLE_EVENTS = %w[
+    consumer.consumed
+  ].freeze
+
+  def instrument(event_id, payload = EMPTY_HASH, &block)
+    return super unless TRACEABLE_EVENTS.include?(event_id)
```

Comment:
Probably not important here because there is only one item but I general for this kind of test we should rely on a hash.

```ruby
TRACEABLE_EVENTS = {
  "consumer.consumed" => true
}.freeze

if TRACEABLE_EVENTS[event_id]
  # ...
end

```

Reply:
It was also copied from the doc. For simple list like this one, I  will keep the current implementation (just a matter of personal preference to be honest :D).
I'm not even sure we will monitor other kind of events in a neat future...